core/src/main/java/org/apache/iceberg/hadoop/HadoopCatalog.java

```diff
@@ -200,7 +211,108 @@ public void renameTable(TableIdentifier from, TableIdentifier to) {
   }
 
   @Override
-  public void close() throws IOException {
+  public void createNamespace(Namespace namespace, ImmutableMap<String, String> meta) {
+    Preconditions.checkArgument(
+        !namespace.isEmpty(),
+        "Cannot create namespace with invalid name: %s", namespace);
+    Preconditions.checkArgument(
+        meta.isEmpty(),
+        "Hadoop Catalog not support metadata %s on namespace: %s", meta, namespace);
```

Comment:
Let's change this error message to "Cannot create namespace %s: metadata is not supported". I think this should also be an `UnsupportedOperationException`, right?